cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255, required=True)
+    description = serializers.CharField(required=True)
+    duration = serializers.IntegerField(required=True)
+
+    def create(self, validated_data: dict) -> Movie:
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance: Movie, validated_data: dict) -> Movie:
+        for field in ("title", "description", "duration"):
+            if field in validated_data:
+                setattr(instance, field, validated_data[field])
```

Comment:
why not instance.title = validated_data[..]?
If some field failed validation, does it mean it'll call this method to update data anyway?